### PR TITLE
Jenkins docs expanded for all the new UI in the plugin

### PR DESCRIPTION
**Pull request by NickJosevski**
The old documentation was focussed on a walk through of building a project, with 2 specific commands executed via the generic shell input.

This change in the documentation showcases each of the 5 actions you can now do with the plugin, including expansion of how to set it up.

The Jenkins plugin has been reworked
 - Supports Spaces
 - Has a lot more UI elements, so there's more screenshots

### CODE REVIEW

File: docs/packaging-applications/build-servers/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Build information is passed to Octopus as a file using a custom format. The reco
 
 :::hint
 **Build Server support**
-The Build Information step is currently available in the official Octopus [TeamCity](/docs/packaging-applications/build-servers/teamcity.md) and [Bamboo](/docs/packaging-applications/build-servers/bamboo.md) plugins. Support for other build servers is coming soon.
+The Build Information step is currently available in the official Octopus [TeamCity](/docs/packaging-applications/build-servers/teamcity.md), [Bamboo](/docs/packaging-applications/build-servers/bamboo.md) and [Jenkins](/docs/packaging-applications/build-servers/jenkins.md) plugins. Support for other build servers is coming soon.
```
Comment (wordlee) [inline docs/packaging-applications/build-servers/index.md]: picky comma :)
```suggestion
The Build Information step is currently available in the official Octopus [TeamCity](/docs/packaging-applications/build-servers/teamcity.md), [Bamboo](/docs/packaging-applications/build-servers/bamboo.md), and [Jenkins](/docs/packaging-applications/build-servers/jenkins.md) plugins. Support for other build servers is coming soon.
```

File: docs/packaging-applications/build-servers/jenkins.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Jenkins
-description: Jenkins can work together with Octopus Deploy to create releases and execute deployments.
+description: Jenkins can work together with Octopus Deploy to create releases, push packages, push build information and execute deployments.
```
Comment (wordlee) [inline docs/packaging-applications/build-servers/jenkins.md]:
```suggestion
description: Jenkins can work together with Octopus Deploy to create releases, push packages, push build information, and execute deployments.
```

File: docs/packaging-applications/build-servers/jenkins.md [hunk 3]
```diff
@@ -14,16 +14,63 @@ Using Jenkins and Octopus Deploy together, you can:
 - Securely deploy your applications with Octopus Deploy across your infrastructure.
 - Fully automate your continuous integration and continuous deployment processes.
 
+Octopus Deploy will take those packages and to push them to development, test, and production environments.
```
Comment (wordlee) [inline docs/packaging-applications/build-servers/jenkins.md]:
```suggestion
Octopus Deploy will take those packages and push them to development, test, and production environments.
```

**Comment by NickJosevski** [comment]
Thanks for the assit @wordlee and catching some silly typos from me.

**Pull request merged**